src/internal/module/build.go

```diff
@@ -27,6 +28,12 @@ func (m Module) UpdateMetadataFile() error {
 // BuildMetadata builds the Metadata for the module by collating the tags from
 // the module repository.
 func (m Module) BuildMetadata() (*Metadata, error) {
+	// Use blacklist from Module struct, fall back to empty if nil
```

Comment:
Above we os.Exit(1) if the blacklist can't be loaded.  Why add the nil check here?

Reply:
Valid. removed.